openmc/statepoint.py

```diff
@@ -451,7 +451,7 @@ def summary(self):
 
     @property
     def with_summary(self):
-        return False if self.summary is None else True
+        return False if self.summary is False else True
```

Comment:
This property doesn't seem terribly useful. Is there a reason to keep it around or should we just get rid of it? I don't see it used anywhere actually.